app/components/videos/videos.service.js

```diff
@@ -1,5 +1,5 @@
 (function () {
-  let VideosService = function (StorageService, $http, YoutubeService, VimeoService, $injector) {
```

Comment:
z tym $injectorem to grubo poleciales. dlaczego? bo musisz z gory wiedziec jakiego serwisu chcesz uzyc, zeby zrobic na nim geta.

powinienes dodac tu wszystkie serwisy jako dependencje, i trzymac je w arrayce, np. services.